build.sh

```diff
@@ -450,18 +450,18 @@ if [ ${BUILD_BE} -eq 1 ]; then
     cp -r -p ${STARROCKS_HOME}/be/output/lib/* ${STARROCKS_OUTPUT}/be/lib/
     # format $BUILD_TYPE to lower case
     ibuildtype=`echo ${BUILD_TYPE} | tr 'A-Z' 'a-z'`
-    if [ "${ibuildtype}" == "release" ] ; then
-        pushd ${STARROCKS_OUTPUT}/be/lib/ &>/dev/null
-        BE_BIN=starrocks_be
-        BE_BIN_DEBUGINFO=starrocks_be.debuginfo
-        echo "Split $BE_BIN debug symbol to $BE_BIN_DEBUGINFO ..."
-        # strip be binary
-        # if eu-strip is available, can replace following three lines into `eu-strip -g -f starrocks_be.debuginfo starrocks_be`
-        objcopy --only-keep-debug $BE_BIN $BE_BIN_DEBUGINFO
-        strip --strip-debug $BE_BIN
-        objcopy --add-gnu-debuglink=$BE_BIN_DEBUGINFO $BE_BIN
-        popd &>/dev/null
-    fi
+    # if [ "${ibuildtype}" == "release" ] ; then
```

Comment:
rm it

Reply:
fixed. sorry, it's to time consuming when building release in this step.